okhttp/src/main/java/io/grpc/okhttp/CipherSuite.java

```diff
@@ -0,0 +1,379 @@
+/*
```

Comment:
This should likely be internal for us as well. We want the copies to be implementation details.